ui/objects/semantic-ui/js/objects/editform/components/VocabularyTreeSelectField.jsx

```diff
@@ -0,0 +1,469 @@
+import React, { useMemo, useState } from "react";
+import { SelectField } from "react-invenio-forms";
+import { useFormConfig } from "@js/oarepo_ui";
+import { useFormikContext, getIn } from "formik";
+import PropTypes from "prop-types";
+import {
+  Breadcrumb,
+  Button,
+  Header,
+  Grid,
+  Input,
+  Icon,
+  Checkbox,
+  Label,
+  Container,
+  Modal,
+  ModalHeader,
+  ModalContent,
+  ModalActions,
+} from "semantic-ui-react";
+import { processVocabularyItems } from "@js/oarepo_vocabularies";
+import { useTranslation } from "react-i18next";
+
+export const VocabularyTreeSelectField = ({
+  fieldPath,
+  multiple,
+  optionsListName,
+  helpText,
+  placeholder,
+  category,
```

Comment:
Category shouldn't be a prop of this component, in other repositories, there's no category at all